src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/locations/LocationService.kt

```diff
@@ -117,7 +115,9 @@ class LocationService(
     val location = agencyInternalLocationRepository.findByIdOrNull(locationId)
       ?: throw NotFoundException("Location with id=$locationId does not exist")
 
-    if (!location.active) {
+    if (deactivateRequest.force) {
+      log.info("Force-deactivating already inactive location $location")
```

Comment:
should this be in the `!location.active` if block?  Otherwise you'll get the message even if the location is active

Reply:
whoah good point! I wanted to see a message when force was true though so its the message that is wrong